lib/index.js

```diff
@@ -159,7 +175,14 @@ function getPolyfillString(options) {
 					}
 				});
 				if (builtPolyfillString) {
-
+					return streamToString(
+						browserify(intoStream(builtPolyfillString))
```

Comment:
Browserify is done on the server for each request, the reason it is not done during build for each polyfill is to enable deduplication of common required modules.